code/optimize/src/main/java/com/adobe/marketing/mobile/optimize/Optimize.java

```diff
@@ -327,4 +327,77 @@ private static void failWithError(final AdobeCallback<?> callback, final AdobeEr
             callbackWithError.fail(error);
         }
     }
+
```

Comment:
Please add java docs for the new public API

Reply:
Java docs are still missing

Reply:
done @spoorthipujariadobe kindly have a look and let me know if you've any suggestion since this is my first time to update the javadoc